.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-latest]
+        os: [ubuntu-16.04, ubuntu-18.04]
```

Comment:
Is there a reason why ubuntu-latest was removed?

Reply:
ubuntu-latest is just an alias for ubuntu-18.04 at the moment.